docs/custom-tests/downloading-extensions-and-tests.md

```diff
@@ -0,0 +1,124 @@
+# Downloading extensions and tests
+
+QIT automatically downloads the extension you are testing **(SUT)** and any additional plugins, dependencies, and test tags. By default, QITc fetches the latest stable versions from WooCommerce.com for paid extensions and WordPress.org for free plugins.
+
+## The extension under test
+
+The main extension you are testing must be a product in the WooCommerce.com marketplace and associated with your account. The account used during `qit connect` **must be the maintainer of this extension**. This can be a product submission or a published product.
+
+For more details, see [Authenticating with QIT](../installation-setup/authenticating.md).
+
+## Downloading paid extensions
+
+Paid extensions require authentication. Similar to the SUT, you **must be the maintainer of the paid extension** you want to include in your test. If you do not own it, you must provide a local source (for example, a ZIP file containing the extension).
```

Comment:
Minor nit, but I think let's focus on `maintain` instead of own for terminology:

```suggestion
Paid extensions require authentication. Similar to the SUT, you **must be the maintainer of the paid extension** you want to include in your test. If you do not maintain it, you must provide a local source (for example, a ZIP file containing the extension).
```